specs-go/config.go

```diff
@@ -70,3 +71,9 @@ type IntelRdt struct {
 	EnableCMT     bool   `json:"enableCMT,omitempty"     yaml:"enableCMT,omitempty"`
 	EnableMBM     bool   `json:"enableMBM,omitempty"     yaml:"enableMBM,omitempty"`
 }
+
+// NetDevice represents an OCI [Linux]NetDevice to be added to the OCI Spec.
```

Comment:
Did we want to name this `LinuxNetDevice`? I'm ok to have parity with the OCI spec though.

Reply:
Renamed.